tests/usbredir_test.go

```diff
@@ -77,7 +77,7 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 			vmi = libvmops.RunVMIAndExpectLaunch(vmi, 90)
 		})
 
-		It("should fail to connect to VMI's usbredir socket", func() {
+		It("should fail to connect to VMI's usbredir socket", decorators.Conformance, func() {
```

Comment:
Unfortunately these tests are not stable, maybe we want make issue and follow up later?